specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerAppsJob_CreateorUpdate.json

```diff
@@ -0,0 +1,207 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppsJobName": "testcontainerAppsJob0",
+    "api-version": "2022-11-01-preview",
+    "containerAppsJobEnvelope": {
+      "location": "East US",
+      "properties": {
+        "environmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
+        "configuration": {
+          "replicaTimeout": 10,
+          "replicaRetryLimit": 10,
+          "manualTriggerConfig": {
+            "replicaCount": 1,
```

Comment:
`replicaCount` should be `>= parallelism`

Reply:
@anthonychu this was modified as per the following suggestion - https://github.com/Azure/azure-rest-api-specs/pull/21846/files/feaac967641b198543cd57b4fb2644b3b3a80a18#r1048685332

@trajkobal requesting your input on this.

Reply:
It can go both ways. 

- If the replica count (completions) is less than parallelism (e.g. you execute 2 parallel replicas and only need one of them to complete, then the job is done when one replica finishes. This is more like providing redundancy.
- If the replica count is more than parallelism then it will repeat replica count / parallelism number of times. i.e run 10 replicas 2 at a time way